src/main/java/com/dev/theater/service/impl/MovieSessionServiceImpl.java

```diff
@@ -1,18 +1,23 @@
 package com.dev.theater.service.impl;
 
 import com.dev.theater.dao.MovieSessionDao;
-import com.dev.theater.library.Inject;
-import com.dev.theater.library.Service;
 import com.dev.theater.model.MovieSession;
 import com.dev.theater.service.MovieSessionService;
 import java.time.LocalDate;
 import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
 @Service
 public class MovieSessionServiceImpl implements MovieSessionService {
-    @Inject
+
     private MovieSessionDao movieSessionDao;
```

Comment:
it is better to make `final` all the dependencies that we autowire on service and dao layers